unime/src/routes/badges/[id]/+page.svelte

```diff
@@ -88,6 +77,7 @@
       {#if credentialLogoUrl}
         <img
           src={credentialLogoUrl}
+          alt=""
```

Comment:
Are we providing `alt` text here for the `img`?

Reply:
Duplicate of https://github.com/impierce/identity-wallet/pull/115#discussion_r1492391882